src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -36,6 +38,16 @@
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
+    /**
+     * Parses user input into command for execution.
+     *
+     * @return the command based on the user input
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public Command parseCommand() throws ParseException {
+        return parseCommand();
+    }
```

Comment:
unused method? 
There is an existing method for this. parseCommand(String userInput)